src/main/java/org/alliancegenome/curation_api/model/entities/AffectedGenomicModel.java

```diff
@@ -55,4 +52,13 @@ public class AffectedGenomicModel extends GenomicEntity {
 	@JsonView({ View.FieldsAndLists.class, View.GeneDetailView.class })
 	private List<ConstructGenomicEntityAssociation> constructGenomicEntityAssociations;
 
+	@IndexingDependency(reindexOnUpdate = ReindexOnUpdate.SHALLOW)
+	@FullTextField(analyzer = "autocompleteAnalyzer", searchAnalyzer = "autocompleteSearchAnalyzer")
+	@KeywordField(name = "synonyms_keyword", aggregable = Aggregable.YES, sortable = Sortable.YES, searchable = Searchable.YES, normalizer = "sortNormalizer")
+	@JsonView({ View.FieldsAndLists.class, View.AffectedGenomicModelView.class  })
```

Comment:
🚫 **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.SingleSpaceSeparatorCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Use a single space to separate non-whitespace characters.
<!-- __reviewdog__:ChBkM2IxMDcyNmJhYzc4YjZmEgpjaGVja3N0eWxl -->